pandas/core/computation/expr.py

```diff
@@ -31,7 +35,17 @@ def tokenize_string(source):
         A Python source code string
     """
     line_reader = StringIO(source).readline
-    for toknum, tokval, _, _, _ in tokenize.generate_tokens(line_reader):
+    token_generator = tokenize.generate_tokens(line_reader)
+
+    # Loop over all tokens till a backtick (`) is found.
+    # Then, take all tokens till the next backtick to form a backtick quoted
+    # string.
```

Comment:
if you are using the uuid3 string, then do you still need the BACKTICK token? and if you still need that, then how is the uuid3 helping?

Reply:
After tokenization of the backtick quoted string, the name of the backtick quoted string needs to be changed to get rid of the spaces. This happens in `def _clean_spaces_backtick_quoted_names(tok)` of `pandas/core/computation/expr.py`. In that function _remove_spaces_column_names(tokval) is called which then uses the uuid3 method to create a unique string. The uuid3 method was just to make sure "column name" and "column_name" are never mixed up. Since the spaces are replaced by underscore, but when this replacement happens the uuid3 string is appended to the string. Before I had just a simple string "_BACKTICK_QUOTED_STRING" appended to it. 

This has nothing to do with the tokenization. The tokenization and the string clean up method are two different things, differently implemented. Which one needs improvement?

Reply:
so my original question was whether you could simply substitue a uuid string for the original string (rather than replace spaces and append with a special string). conceptually I like the former, it is unclear i this is possible. that is what I was asking.

Reply:
Yes, that is possible. But if the query has errors, I am afraid the error will be less clear, since the original name is gone.

Reply:
ok, fair enough. let's revert the uuid stuff then; your original way seemed reasonable.

Reply:
Reverted, also removed from compat.